pkgs/development/compilers/nim/default.nix

```diff
@@ -1,10 +1,12 @@
 # based on https://github.com/nim-lang/Nim/blob/v0.18.0/.travis.yml
 
-{ stdenv, lib, fetchurl, makeWrapper, openssl, pcre, readline,
-  boehmgc, sfml, sqlite }:
+{ stdenv, lib, fetchurl, makeWrapper, openssl, pcre, readline
+, boehmgc, sfml, sqlite
+, minimal ? false
```

Comment:
Cross-compiling the minimal nim is currently broken. I think I went to far with dropping dependencies.